### DISCUSSION TITLE
Welcome to crytic-compile Discussions!

### DISCUSSION THREAD

**Discussion by montyly** [crytic/crytic-compile#395] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Saniabuh** [comment]
Please how can I address this code

TypeError: "send" and "transfer" are only available for objects of type "address payable", not "address".
  --> Test.sol:22:9:
   |
22 |         msg.sender.transfer (reward_TOD24);
   |         ^^^^^^^^^^^^^^^^^^^

**Reply by talfao** [reply]
You have the answer in the error message.
